### ISSUE TITLE
0.9 Release Cluster Write Errors

### ISSUE THREAD

**Issue by Jhors2** [influxdb/influxdb#2960]
I am receiving the following message every time I attempt to write points into my DB:
```[write] 2015/06/12 06:57:13 write failed for shard 3 on node 1: error code 1: write shard 3: field type conflict: input field "value" is type int64, already exists as type %!s(influxql.DataType=1)```

```[write] 2015/06/12 06:57:13 write failed for shard 3 on node 2: error code 1: write shard 3: field type conflict: input field "value" is type int64, already exists as type %!s(influxql.DataType=1)```

The interesting thing is that all of my values are scraped before writing to make sure some decimal value has not returned.  I also dropped my database entirely, created a new one then manually instantiated my measurements with int type to attempt to prevent float type from being set as follows:
```curl "http://influx1.internal:8086/write?db=testdb" -d 'cpu value=10 1000000000001'```

After performing the above, whenever a write is made to another node it is for some reason created with float type from what I can tell.

Any thoughts here?

**Comment by jwilder** [comment]
@Jhors2 What do you see if you query the data?  Does it return ints or floats?

**Comment by Jhors2** [comment]
@jwilder The queries (I performed one across each measurement) always return ints.
Example:
``` ./influx -database 'monocle' -execute 'select * from as_path_bps limit 5;'

[5:07 PM] Eric Cables: name: cpu
tags: server=web01,site=SFO
time                value
----                -----
2015-06-12T13:46:07.972194Z    1000516
2015-06-12T13:55:08.000088Z    1000516
2015-06-12T13:56:07.983637Z    1000516
2015-06-12T13:58:08.077041Z    1000516
2015-06-12T14:01:08.347279Z    1000516```

**Comment by jwilder** [comment]
Would you be able to post a sequence of steps to recreate this?  E.g. create database, RP, your initial write and then some sample writes that fail.  Not sure what's going on.

**Comment by Jhors2** [comment]
Sure:
```drop database testdb;```
```create database testdb;```
```CREATE RETENTION POLICY infrep ON testdb DURATION inf REPLICATION 3 default;```
```curl "http://influx1.internal:8086/write?db=testdb&precision=s&consistency=quorum" -d 'hose_as_output_bps,hose_id=xxxx:31150,as_type=peer_dst_as,as_num=39405,protocol=ipv4 value=6632243 1434123302'```
```curl "http://influx1.internal:8086/write?db=testdb&precision=s&consistency=quorum" -d 'hose_as_output_bps,hose_id=xxxx:31150,as_type=peer_dst_as,as_num=43100,protocol=ipv4 value=253405 1434123302'```

**Comment by jwilder** [comment]
Thanks @Jhors2.   I was able to reproduce the issue.  Fix is in #2967
